src/django/api/tiler.py

```diff
@@ -19,28 +22,91 @@ def get_facilities_vector_tile(layer, z, x, y, params=None):
             )
         ) """
 
-    facilities = Facility.objects.filter(
+    base_qs = Facility.objects.filter(
         location__within=Polygon.from_bbox((
             tile_bounds.west, tile_bounds.south,
             tile_bounds.east, tile_bounds.north))
-    ).extra(
-        select={
-            'location': mvt_geom_query.format(
-                xmin=tile_bounds.west,
-                ymin=tile_bounds.south,
-                xmax=tile_bounds.east,
-                ymax=tile_bounds.north,
-                buffer_distance=buffer_distance
-            )
-        }
-    ).values('location', 'id')
+    )
+
+    free_text_query = request.query_params.get(FacilitiesQueryParams.Q,
```

Comment:
Reimplementing these filters in tiler.py would eliminate a major benefit of using our Django app as a tiler, which is reusing all of the the query parameter processing logic. We should consider creating a shared function that converts a query params dictionary into a facilities queryset. To render sidebar search results, we just use that queryset as is. To render a tile, we add the additional bbox filter clause to that queryset, get `.values()`, then post process the SQL we obtain from it.